Library/Formula/postgresql.rb

```diff
@@ -77,8 +82,16 @@ def install
   end
 
   def post_install
-    unless File.exist? "#{var}/postgres"
-      system "#{bin}/initdb", "#{var}/postgres"
+    system "pkill postgres || true"
+
+    if (var/"postgres").exist?
+      mv var/"postgres", var/"postgres-#{version_suffix}"
```

Comment:
Probably the best thing for us to do is:

1. Rename the existing data to the correctly suffix versioned directory
2. If the version has changed, copy the data to the new suffix versioned directory and run `pg_upgrade`
3. Tell the user what we did.